lib/src/main/kotlin/bstrees/balancers/RBBalancer.kt

```diff
@@ -3,8 +3,79 @@ package bstrees.balancers
 import bstrees.nodes.RBNode
 
 class RBBalancer<T : Comparable<T>> : TreeBalancer<T, RBNode<T>> {
+    private fun getColor(node: RBNode<T>?) = node?.color ?: RBNode.Color.Black
+
+    private fun isRed(node: RBNode<T>?) = getColor(node) == RBNode.Color.Red
+
+    private fun isBlack(node: RBNode<T>?) = !isRed(node)
+
+    private fun getSibling(node: RBNode<T>): RBNode<T>? {
+        val parent = node.parent ?: return null
+        return if (parent.left == node) parent.right
+        else parent.left
+    }
+
+    private fun getUncle(node: RBNode<T>): RBNode<T>? {
+        val parent = node.parent ?: return null
+        return getSibling(parent)
+    }
+
+    private fun rotateRight(node: RBNode<T>): RBNode<T> {
+        val parent = node.parent
+        val wasChild = node.left ?: throw IllegalArgumentException("Incorrect argument for function rotateRight()")
```

Comment:
Description passed to the exception doesn't tell anything about the problem.
It should describe why exactly the argument is incorrect